util/cli_ui.go

```diff
@@ -131,8 +131,14 @@ func (u *ColorizeUi) parseJsonPath(input interface{}, template string) (interfac
 		return nil, errors.New(fmt.Sprintf("Error parsing value from input %v using template %s: %v ", input, template, err))
 	}
 
-	if values != nil && len(values) > 0 && len(values[0]) > 0 {
-		return values[0][0].Interface(), nil
+	if values != nil && len(values) > 0 {
+		valueStrings := []string{}
+		for idx := range values {
+			for val := range values[idx] {
+				valueStrings = append(valueStrings, fmt.Sprintf("%v", values[idx][val].Interface()))
+			}
+		}
+		return strings.Join(valueStrings, ""), nil
```

Comment:
Given the example expression you mentioned `"jsonpath={[]..id},{[]..application}"`, what is the expected output  for an application with multiple pipelines?

Reply:
With `jsonpath={[*]..id},{[*]..application}` it would be something like `uuid1uuid2uuid3,name1name2name3`.

Without this change the output would be:
`uuid1`

Reply:
Thoughts on adding a separator? Kubectl seems to use a space between values.

Reply:
Yeah, I'm getting more partial to their way of doing it the more I use this patch. I'll make some changes